integration_tests/suite/test_confgen.py

```diff
@@ -0,0 +1,159 @@
+import subprocess
+import pytest
+from wazo_test_helpers import asset_launching_test_case
+import pathlib
+import sys
+from typing import List, Iterable
+
+
+INTERNAL_CONFGEND_PORT = 8669
+DEFAULT_CONFGEND_CLIENT_TIMEOUT = 10
+DEFAULT_TIMEOUT = 10
+
+@pytest.fixture(scope="class")
+def pjsip_void_conf(request):
+    with open("suite/pjsip_void.conf") as file:
+        conf = file.readlines()
+
+    request.cls.pjsip_void_conf = conf
+
+
+def normalize_lines(line_stream: Iterable[str]):
+    for line in line_stream:
+        line = line.strip()
+        if line:
+            yield line
+
+
+def lines(text: str):
+    return text.splitlines()
+
+
+def run(cmd: List[str], timeout: int = DEFAULT_TIMEOUT) -> subprocess.CompletedProcess:
+    try:
+        result = subprocess.run(
+            cmd, capture_output=True, text=True, check=True, timeout=timeout
+        )
+    except subprocess.CalledProcessError as ex:
+        print(ex.stderr)
+        raise
+    return result
+
+
+class BaseTestCase(asset_launching_test_case.AssetLaunchingTestCase):
+    service = "confgend"
+    assets_root = pathlib.Path(__file__).parent.parent / "assets"
+    asset = "base"
+
+    def docker_exec(
+        cls, cmd: List[str], service_name: str = None
+    ) -> subprocess.CompletedProcess:
+        service_name = service_name or cls.service
+        docker_command = [
+            "docker-compose",
+            *cls._docker_compose_options(),
+            "exec",
+            service_name,
+        ] + cmd
+        try:
+            completed = run(docker_command)
+        except subprocess.CalledProcessError as ex:
+            print(ex.stderr, file=sys.stderr)
+            raise
+        return completed
+
+    @classmethod
+    def client(cls, args):
+        # going through host local port-map
+        host = "127.0.0.1"
+        timeout = DEFAULT_CONFGEND_CLIENT_TIMEOUT
+        return run(
+            [
+                "wazo-confgen",
+                "--port",
+                str(cls.service_port(INTERNAL_CONFGEND_PORT)),
+                "--host",
+                host,
+                "--timeout",
+                str(timeout),
+                *args,
+            ],
+            timeout,
+        )
+
+
+@pytest.mark.usefixtures("pjsip_void_conf")
```

Comment:
Why is it useful to use pytest fixture in this case?
We only have one test that use asterisk/pjsip.conf
Maybe for now it would be clearer to write code directly in the test (or use fixture only for specific test)